cli/tests/pcluster/aws/test_ec2.py

```diff
@@ -334,10 +334,14 @@ def test_run_instances_dryrun(boto3_stubber, error_code, raise_exception):
         Ec2Client().run_instances(**kwargs)
 
 
-def get_describe_subnets_mocked_request(subnets, state):
+def get_describe_subnets_mocked_request(subnets, state, avail_zones: dict):
     return MockedBoto3Request(
```

Comment:
I see what you mean and I like the idea of the namedtuple.

I had a quick try and realized that this would probably require a bit of thought about the `test_describe_subnets_cache` function, which relies on the fact that you can do multiple requests for the same subnet (e.g. with different states).

If it's fine to you I would leave it as it is for now.